data-pipeline/tests/pipeline/test_new_clinvar_xml_format.py

```diff
@@ -0,0 +1,293 @@
+import gzip
```

Comment:
I notice that we introduce this file here and then delete it a few commits down the line. Generally speaking it's best not to introduce something into a PR and then delete it later, but rather to rebase the branch before opening the PR so that the stuff in question just never appears in the PR in the first place.

Now in this case, I think we may actually want to save this script, since presumably we're going to want to test this process again at some point in the future, but I'll leave it to your judgment about how likely that is. If we do save it, it would be good to get rid of the `_NEW_foo` function names and other rough edges.